src/mame/drivers/vsmjtria.cpp

```diff
@@ -476,6 +488,26 @@ void vsmjtria_state::machine_start()
 	save_item(NAME(m_keyboard_cmd));
 }
 
+void vsmjtria_state::machine_reset()
+{
+	if (m_sw2->read() & 1)
+	{
+		// TODO: there's no setter in nvram_device that directly flushes contents for this case scenario
+		// we currently workaround by directly inject to space address
+		const u16 nvram_size = 0x800;
+		address_space &space_main = m_cpu[0]->space(AS_PROGRAM);
+		address_space &space_sub = m_cpu[1]->space(AS_PROGRAM);
+
+		for (int i = 0; i < nvram_size; i++)
+		{
+			space_main.write_byte(0xa000 + i, 0x00);
+			space_sub.write_byte(0xa000 + i, 0x00);
+		}
+
+		logerror("%s: flush NVRAM contents\n", machine().describe_context());
```

Comment:
This is always called from the scheduler – it will never show useful context in `describe_context`.

Reply:
As far as I know, you can "force" a NVRAM clear via external methods on:

-  Sega Saturn (by removing the cr2032 battery from the back lid, cfr. `m_smem` behaviour in `src\mame\devices\smpc.cpp`, which also incidentally clears the backup RAM contents from a separate bus) 
- PC98HA (from a port C bit reading which is WIP about how to readback, software definitely expects that bit to be some sort of NVRAM NG -> init to default check)